sdk/python/.pylintrc

```diff
@@ -159,7 +159,8 @@ disable=print-statement,
         broad-except,
         no-self-use,
         unused-import,
-        unsubscriptable-object
+        unsubscriptable-object,
+        line-too-long
```

Comment:
This feels like one we may not want to turn off - I think we do for the most part want to maintain discipline around this?

Reply:
i find this rule just enormous frustrating.  At 100, it feels like even basic stuff just constantly hits it.  It's particularly annoying in libraries where we're trying to keep consistency.  in this case, i have a bunch of statements in a row that would be good to keep totally in sync, having to wrap one at the expense of the others just hurts readability for me.  

I feel like we can trust people to make reasonable judgements on line length TBH.